test/multi.html

```diff
@@ -43,6 +44,20 @@
     </template>
   </test-fixture>
 
+  <!--
+  NOTE(cdata): Enable test-fixture when polymer/polymer#2495 is resolved
+  -->
+  <!--<test-fixture id="repeatedItems">
```

Comment:
Theory: enable the test, but expect it to fail always in Safari? WDYT?
(at least then we'd get slighty more coverage)